x-pack/plugins/fleet/public/applications/fleet/sections/agent_policy/create_package_policy_page/components/steps/components/package_policy_input_stream.tsx

```diff
@@ -53,19 +61,22 @@ export const PackagePolicyInputStreamConfig: React.FunctionComponent<{
     // Errors state
     const hasErrors = forceShowErrors && validationHasErrors(inputStreamValidationResults);
 
-    const requiredVars: RegistryVarsEntry[] = [];
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-    const advancedVars: RegistryVarsEntry[] = [];
+    const [requiredVars, advancedVars] = useMemo(() => {
```

Comment:
not related to that PR but I add to put this in a `useMemo` otherwise eslint was complaining